client/i18n/locales/english/intro.json

```diff
@@ -7485,10 +7491,10 @@
       }
     },
     "blocks": {
-      "lecture-understanding-the-http-request-response-model": {
-        "title": "Understanding the HTTP Request-Response Model",
+      "lecture-working-with-nodejs-and-event-driven-architecture": {
+        "title": "Working with NodeJS and event driven architecture",
         "intro": [
-          "Learn the fundamentals of how web communication works through the HTTP request-response model, explore different types of web assets and responses, and understand how forms handle data submission using various HTTP methods."
+          "Learn about Node.js core libraries, how to install Node.js on your computer, and the advantages and disadvantages of using Node.js on the backend."
```

Comment:
the http request response model block is an existing block, I don't believe the intro should be deleted